src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -307,12 +307,24 @@ private function getProperties(UploadedFile $uploadedFile)
         $properties = [];
 
         try {
-            // if the file is a video we add the duration
+            // if the file is a video we add the properties related to it
             if (fnmatch('video/*', $mimeType)) {
+                // Duration
                 $properties['duration'] = $this->ffprobe->format($uploadedFile->getPathname())->get('duration');
+
+                // Dimensions
+                try {
+                    $dimensions = $this->ffprobe->streams($uploadedFile->getPathname())->videos()->first()->getDimensions();
+                    $properties['width'] = $dimensions->getWidth();
+                    $properties['height'] = $dimensions->getHeight();
+                } catch (InvalidArgumentException $e) {
+                    // Exception is thrown if the video stream could not be obtained
+                } catch (RuntimeException $e) {
```

Comment:
Isn't there a more specific exception that could be caught? This might hide all different kind of exceptions...

Reply:
Unfortunately, I don't think so. Looking at the ffmpeg lib, it seems that `RuntimeException` is what the different methods used to get the video's dimensions throw:
- https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/d1a281f30ffbee3110a58b1d41644ed0782f505b/src/FFMpeg/FFProbe.php#L238
- https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/d1a281f30ffbee3110a58b1d41644ed0782f505b/src/FFMpeg/FFProbe.php#L263
- https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/d1a281f30ffbee3110a58b1d41644ed0782f505b/src/FFMpeg/FFProbe/DataMapping/Stream.php#L46

We could also catch the `LogicException` from `FFProbe::getDimensions` but it should not be triggered as we only query video streams. In any case, this would not suffice to be more precise on exception catching.

If you think this is too broad, I can try formatting the code another way. I'm open to suggestions. Thanks for your review!

Reply:
Well, the only thing we could do is check the message of the exception, and that's also not nice... I guess we let it that way in this case.